.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @ro-tex @ChrisSchinnerl
```

Comment:
PJ owns the entire abuse stack. He should be code owner here.
I don't mind whether the second code owner is me or Chris.